testing/src/main/java/net/consensys/linea/testing/BytecodeRunner.java

```diff
@@ -107,7 +107,7 @@ public void run(Wei senderBalance, Long gasLimit, List<ToyAccount> additionalAcc
         ToyTransaction.builder()
             .sender(senderAccount)
             .to(receiverAccount)
-            .value(Wei.ONE)
+            .value(Wei.of(69))
```

Comment:
:)

Reply:
Mostly to disambiguate with other 1's in the trace 🙂.